content/en/code_coverage/_index.md

```diff
@@ -8,12 +8,8 @@ algolia:
   tags: ["code coverage"]
 ---
 
-{{< site-region region="gov" >}}
```

Comment:
Double-checking whether the "gov" site warnings were removed intentionally (here and in the setup page)